rumbl/priv/repo/seeds.exs

```diff
@@ -9,3 +9,11 @@
 #
 # We recommend using the bang functions (`insert!`, `update!`
 # and so on) as they will fail if something goes wrong.
+
+alias Rumbl.Repo
+alias Rumbl.Category
+
+for category <- ~w(Action Drama Romance Comedy Sci-fi) do
+  Repo.insert!(%Category{name: category}) ||
+    Repo.insert!(%Category{name: category})
```

Comment:
@shouston3 good shout. please feel free to edit the file in PR. 👍 
`else` I will fix tomorrow morning. 😉